x-pack/legacy/plugins/apm/server/routes/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { indexPatternRoute } from './index_pattern';
+import {
+  errorDistributionRoute,
+  errorGroupsRoute,
+  errorsRoute
+} from './errors';
+import { createApi } from './create_api';
+
+const api = createApi()
+  .add(indexPatternRoute)
+  .add(errorDistributionRoute)
+  .add(errorGroupsRoute)
+  .add(errorsRoute);
```

Comment:
Perhaps this should be inside a function call so merely requiring the file doesn't have any side-effects?

Reply:
Yeah, will change!